src/netframework/index.ts

```diff
@@ -324,7 +324,14 @@ function replaceAttribute(content: string, name: string, regEx: string, value: s
             });
         }
     } else {
-        content = content.replace(new RegExp(`${name}\\s*\\w*\\(${regEx}\\)`, 'gi'), `${name}("${value}")`);
+
+        let attribute = `${name}("${value}")`;
+
+        if(file.endsWith('.cs') && name === "AssemblyDescription" && isMultiline(value)) {

```

Comment:
@BMuuN Why only C# files? The same @ symbol is used for .VB files

Reply:
The verbatim string literal is not supported in VB